src/main/kotlin/lotto/Controller.kt

```diff
@@ -0,0 +1,79 @@
+package lotto
+
+import lotto.constant.COMMA
+import lotto.model.Lotto
+import lotto.model.LottoGenerator
+import lotto.model.LottoResult
+import lotto.view.InputView
+import lotto.validator.*
+import lotto.view.OutputView
+
+class Controller {
+
+    val input = InputView()
+    val output = OutputView()
+    val inputValidator = InputValidator()
+    val lottoGenerator = LottoGenerator()
+
+    fun start() {
+        //LottoResult 객체 초기화
+        LottoResult.reset()
+
+        val money = getMoney()
+
+        val randomLotto = getRandomLotto(money)
+        output.displayLotto(randomLotto)
+
+        val winningNum = Lotto(getWinningNum())
+
+        val bonusNum = getBonusNum(winningNum)
+
+        winningNum.getResult(money, bonusNum, randomLotto)
+        output.displayResult()
+    }
+
+    fun getMoney(): Int {
+        var money: String
+        while (true) {
+            try {
+                money = input.getMoney()
+                inputValidator.isValidMoney(money)
+                return money.replace(COMMA, "").toInt()
+            } catch (e: IllegalArgumentException) {
+                println(e.message)
+            }
+        }
+    }
+
+    fun getRandomLotto(money: Int): List<List<Int>> {
+        return lottoGenerator.generateLotto(money)
+    }
+
+    fun getWinningNum(): List<Int> {
+        var winningNum: String
+
+        while (true) {
+            try {
+                winningNum = input.getWinningNum()
```

Comment:
```suggestion
        while (true) {
            try {
                val winningNum = input.getWinningNum()
```

이렇게 수정하며 winningNum을 불변으로 받아도 될 것 같습니다!
getMoney(), getWinningNum(), getBonusNum() 전부 같은 방식으로 수정 가능할 것 같아요